src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
I would recommend to make `userService` a class member rather than local variable, as it improves flexibility and simplifies testing

Reply:
Fixed. Is it a good practice in general, to inject other classes in this way, or just in this case?

Reply:
Yes, we will learn about this in the future lectures.